src/modules/expense/PriceSection.tsx

```diff
@@ -0,0 +1,33 @@
+import { styled, Theme } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+
+const StyledDiv = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  flex: 'end',
+  flexDirection: 'column',
+  margin: '80px 30px 0 30px',
+}));
+
+const StyledDivPrice = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  flex: 'end',
+  flexDirection: 'column',
+  gap: '12px',
+}));
+
+export const PriceSection = () => {
+  return (
+    <StyledDiv className='styledDiv'>
```

Comment:
You don't need to add className like this to the styled component, remove everywhere where used.
```
    <StyledDiv className='styledDiv'>
```